src/components/CalloutBanner.tsx

```diff
@@ -1,72 +1,11 @@
 import React from "react"
-import styled from "@emotion/styled"
 import { GatsbyImage, IGatsbyImageData } from "gatsby-plugin-image"
+import { Flex, FlexProps, Heading, Image, Text } from "@chakra-ui/react"
 
 import Translation from "./Translation"
 import { TranslationKey } from "../utils/translations"
 
-const StyledCard = styled.div`
-  display: flex;
-  flex-direction: row-reverse;
-  background: ${({ theme }) => theme.colors.layer2Gradient};
-  padding: 3rem;
-  margin: 1rem;
-  margin-top: 6rem;
-  margin-bottom: 10rem;
-  border-radius: 4px;
-
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column;
-    margin-bottom: 1rem;
-    margin: 4rem 2rem;
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    padding: 2rem;
-  }
-`
-
-const Content = styled.div`
-  padding-left: 2rem;
-  flex: 1 0 50%;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin-top: 2rem;
```

Comment:
This one is missing

Reply:
Yes I know but it's pretty weird because as you can see on the picture bellow from [here](https://ethereum.org/en/stablecoins)
![image](https://user-images.githubusercontent.com/43795504/191129514-ab72553a-6bb2-4b03-b720-eadd8af50a50.png)

It seems there is no margin top on the element...

Reply:
Yeah ok, I think I understand what is going on. There seems to be too much overriding of styles in this component.

I think a more elegant way of fixing this is by removing all the default margin styles this component has and delegate that aspect of the styles to the consumer.

Later I'll jump into this PR and make some adjustments related to that. I'll:
- remove all the margins from the component container
- refactor all the consumers to pass the correct margins for each case

With this approach we will remove that "responsability" from the component and make it simpler.